src/cosmosdb-continuous-backup/README.md

```diff
@@ -0,0 +1,102 @@
+# Microsoft Azure CLI 'cosmosdb-continuous-backup' Extension #
+
+This package is for the 'cosmosdb-continuous-backup' extension.
+This package provides commands to
+
+- Create Azure CosmosDB continuous backup accounts
+- List the different versions of databases and collections that were modified
+- Trigger a point in time restore on the Azure CosmosDB continuous mode backup accounts.
+- Update the backup interval and backup retention of periodic mode bbackup accounts.  
```

Comment:
nit: typo.  also the last two sentences end with a period but the first two don't.  should keep them consistent

Reply:
Fixed the typo and removed the periods.